Package/gulpfile.js

```diff
@@ -0,0 +1,117 @@
+const util = require('util');
+const fs = require('fs');
+const spawn = require('child_process').spawn;
+const readdirAsync = util.promisify(fs.readdir);
+const log = require('fancy-log');
+const del = require('del');
+const gulp = require('gulp');
+const parseArgsStringToArgv = require('string-argv').parseArgsStringToArgv;
+
+async function exec(command, options = {}, logCommand = true) {
+  if (logCommand) {
+    log(command);
+  }
+
+  const arguments = parseArgsStringToArgv(command);
+  command = arguments.shift();
+
+  await new Promise((resolve, reject) => {
+    const process = spawn(command, arguments, options);
+
+    process.on('close', code => code ? reject(code) : resolve());
+
+    process.stdout.on('data', data => {
+      const message = data.toString();
+      if (message !== '\n') {
+        log(message);
+      }
+    });
+
+    process.stderr.on('data', data => {
+      log.error(data.toString());
+    })
+  });
+}
+
+const clean = async () => {
+  await del('Assembled', {force:true});
+};
+
+const makeXCodeProj = async () => {
+  await exec('mkdir -p iOS/Build');
+  await exec('cmake -G Xcode -DCMAKE_TOOLCHAIN_FILE=../../../Apps/Playground/node_modules/@babylonjs/react-native/submodules/BabylonNative/Dependencies/ios-cmake/ios.toolchain.cmake -DPLATFORM=OS64COMBINED -DENABLE_ARC=0 -DDEPLOYMENT_TARGET=12 -DENABLE_GLSLANG_BINARIES=OFF -DSPIRV_CROSS_CLI=OFF ..', {cwd: 'iOS/Build'});
+};
+
+const buildIphoneOS = async () => {
+  await exec('xcodebuild -sdk iphoneos -configuration Release -project ReactNativeBabylon.xcodeproj -scheme BabylonNative build CODE_SIGNING_ALLOWED=NO', {cwd: 'iOS/Build'});
+};
+
+const buildIphoneSimulator = async () => {
+  await exec('xcodebuild -sdk iphonesimulator -configuration Release -project ReactNativeBabylon.xcodeproj -scheme BabylonNative build CODE_SIGNING_ALLOWED=NO', {cwd: 'iOS/Build'});
+};
+
+const buildIOS = gulp.series(makeXCodeProj, buildIphoneOS, buildIphoneSimulator);
+
+const buildAndroid = async () => {
+  await exec('./gradlew babylonjs_react-native:assembleRelease', {cwd: '../Apps/Playground/android'});
+};
+
+const copyCommonFiles = async () => {
+  return  gulp.src('../Apps/Playground/node_modules/@babylonjs/react-native/package.json')
+    .pipe(gulp.src('../Apps/Playground/node_modules/@babylonjs/react-native/README.md'))
+    .pipe(gulp.src('../Apps/Playground/node_modules/@babylonjs/react-native/*.ts*'))
+    .pipe(gulp.src('react-native-babylon.podspec'))
+    .pipe(gulp.dest('Assembled'));
+};
+
+const copyIOSFiles = async () => {
+  return  gulp.src('../Apps/Playground/node_modules/@babylonjs/react-native/ios/*.h')
+    .pipe(gulp.src('../Apps/Playground/node_modules/@babylonjs/react-native/ios/*.mm'))
+    // This xcodeproj is garbage that we don't need in the package, but `pod install` ignores the package if it doesn't contain at least one xcodeproj. 🤷🏼‍♂️
+    .pipe(gulp.src('iOS/Build/ReactNativeBabylon.xcodeproj**/**/*'))
+    .pipe(gulp.dest('Assembled/ios'));
+};
```

Comment:
Do these need to be async if they don't use await?

Reply:
No, removed.